wiki/Interface/zh.md

```diff
@@ -2,46 +2,52 @@
 outdated: true
 ---
 
-# Interface (中文)
+# 界面
```

Comment:
delete the outdated mark at the top if the translation fully matches the English article, and is up-to-date

Reply:
ok